### PR TITLE
Update bitfinex ticker fields

### PR DESCRIPTION
**Pull request by TonnTamerlan**
Bitfinex tickers can have two different types
[documentation](https://docs.bitfinex.com/reference#rest-public-ticker)

### CODE REVIEW

**Comment by TonnTamerlan** [comment]
https://github.com/knowm/XChange/issues/3262

File: xchange-bitfinex/src/main/java/org/knowm/xchange/bitfinex/v2/dto/marketdata/BitfinexTicker.java [hunk 1]
```diff
@@ -2,104 +2,50 @@
...
-
-  public BigDecimal getLow() {
-    return low;
-  }
-
-  public BigDecimal getAsk() {
-    return ask;
-  }
+  private Object placeHolder0;
+  private Object placeHolder1;
```
Comment (walec51) [inline xchange-bitfinex/src/main/java/org/knowm/xchange/bitfinex/v2/dto/marketdata/BitfinexTicker.java]: do you have some example values this placeholder may have?
if so then please add then to your test

File: xchange-bitfinex/src/main/java/org/knowm/xchange/bitfinex/v2/dto/marketdata/BitfinexTicker.java [hunk 2]
```diff
@@ -2,104 +2,50 @@
...
+  private BigDecimal frrAmountAvailable;
 
   @Override
   public String toString() {
     return "BitfinexTicker{"
         + "symbol='"
         + symbol
         + '\''
+        + ", frr="
+        + frr
```
Comment (walec51) [inline xchange-bitfinex/src/main/java/org/knowm/xchange/bitfinex/v2/dto/marketdata/BitfinexTicker.java]: if we're going with lombok we can just use `@ToString` and remove this method

File: xchange-bitfinex/src/main/java/org/knowm/xchange/bitfinex/v2/dto/marketdata/BitfinexTicker.java (hunk 1 above)
Comment (TonnTamerlan) [reply xchange-bitfinex/src/main/java/org/knowm/xchange/bitfinex/v2/dto/marketdata/BitfinexTicker.java]: these placeholders are reserved for the future and they are always null

Comment (TonnTamerlan) [reply xchange-bitfinex/src/main/java/org/knowm/xchange/bitfinex/v2/dto/marketdata/BitfinexTicker.java]: @walec51 these placeholders are reserved for the future and they are always null

File: xchange-bitfinex/src/main/java/org/knowm/xchange/bitfinex/v2/dto/marketdata/BitfinexTicker.java (hunk 2 above)
Comment (TonnTamerlan) [reply xchange-bitfinex/src/main/java/org/knowm/xchange/bitfinex/v2/dto/marketdata/BitfinexTicker.java]: @walec51 fixed

**Comment by TonnTamerlan** [comment]
@walec51 Could you please merge this pull request

**Pull request merged**